### ISSUE TITLE
Sort causes heap corruption in Windows

### ISSUE THREAD

**Issue by TheDan64** [ritchie46/polars#98]
I'm trying to sort a (9 x ~8000) df, and I get the process exiting with `(exit code: 0xc0000374, STATUS_HEAP_CORRUPTION)` (windows 10, rust 1.48.0-nightly (ef663a8a4 2020-09-30), polars v0.6.0).

It doesn't seem to matter whether I use `sort`, or `sort_in_place`. I've also tried sorting two different kinds of columns: timestamps and u32s, with identical outcomes. I watched my RAM usage and I still had 16GB free while it was running, so I don't think it's OOM.

**Comment by TheDan64** [comment]
I can't reproduce it on Linux with the same nightly version

**Comment by ritchie46** [comment]
Thanks for reporting this. Have you got a minimal example with which you can reproduce this on windows? I have tried a very minimal example and couldn't find any corruption problems in valgrind.

```rust
use polars::prelude::*;

fn main() {
    let s0 = Series::new("days", [0, 1, 2].as_ref());
    let s1 = Series::new("temp", [22.1, 19.9, 7.].as_ref());
    let mut df = DataFrame::new(vec![s0, s1]).unwrap();
    df.sort_in_place("temp", false).unwrap();
    println!("{:?}", df);
}

```

**Comment by ritchie46** [comment]
Fixed in 7fafb0cb2fed1659dbf8977ede3503b9c0800201 and release [0.6.1](https://crates.io/crates/polars). All tests succeed in [Windows](https://github.com/ritchie46/polars/actions/runs/286932289). So I think this will solve your issue as well :)

**Comment by TheDan64** [comment]
Thank you!
